packages/grafana-ui/src/components/ClipboardButton/ClipboardButton.tsx

```diff
@@ -60,19 +61,20 @@ export function ClipboardButton({
     }
   }, [getText, onClipboardCopy, onClipboardError]);
 
+  const copiedText = t('clipboard-button.inline-toast.success', 'Copied');
   return (
     <>
       {showCopySuccess && (
         <InlineToast placement="top" referenceElement={buttonRef.current}>
-          <Trans i18nKey="clipboard-button.inline-toast.success">Copied</Trans>
+          {copiedText}
         </InlineToast>
       )}
 
       <Button
         onClick={copyTextCallback}
         icon={icon}
         variant={showCopySuccess ? 'success' : variant}
-        aria-label={showCopySuccess ? 'Copied' : undefined}
+        aria-label={showCopySuccess ? copiedText : ariaLabel}
```

Comment:
If an `aria-label` was being provided to the button, it was always being set, but now I've made it use the passed label, unless a copy was just successful - in that case, it goes `label from props` -> `copied` -> back to `label from props`